VideoWeb/VideoWeb/ClientApp/src/sass/abstracts/_variables.scss

```
@@ -1,2 +1,12 @@
+// 1024px
 $breakpoint-large: 64em;
+// 768px
 $breakpoint-medium: 48em;
+// 640px
+$breakpoint-small-medium: 40.0625em;
+// 480px
+$breakpoint-small: 30em;
+// 360px
+$breakpoint-extra-small: 22.5em;
+// 890px
+$landscape-large: 55.625em;
```

I think there's too many breakpoints here now. Where would 22.5em/360px be relevant? I think we can have the smallest breakpoint as 480px.

I also think we can get rid of the 890 and just use 1024.

```suggestion
// 1024px
$breakpoint-large: 64em;
// 768px
$breakpoint-medium: 48em;
// 640px
$breakpoint-small: 40.0625em;
// 480px
$breakpoint-extra-small: 30em;
```